scripts/thin_airfoil_unsteady.py

```diff
@@ -13,39 +13,53 @@ def compute_gam(body, vort):
     dgam = gam / ds
     xvort = vort.positions
     s = np.sqrt(xvort[:,0]**2 + xvort[:,1]**2)
-    return s, dgam
+    return s, gam, dgam
 
 num_points = 32
-angle = 2 # degrees
-body = flat_plate(num_points)
-body = TransformedBody(body, angle=angle)
+body = TransformedBody(flat_plate(num_points), angle=alpha_deg)
+#body = TransformedBody(naca_airfoil('0012',num_points), angle=alpha_deg)
 bound = BoundVortices(body)
 
 # unsteady simulation
 dt = 0.5
 Uinfty = (1,0)
 num_steps = 100
-# stepper = ExplicitEuler(dt, Uinfty, bound)
 stepper = RungeKutta2(dt, Uinfty, bound)
 
 for i in range(num_steps):
     stepper.advance()
 
-vort = stepper.wake.positions
 print("Using %s timestepper" % stepper.__class__.__name__)
 print("Total circulation: %f" % stepper.wake.circulation)
-print("After %d steps, last shed vortex: %f" %
+print("After %d steps, last shed vortex has strength: %f" %
       (num_steps, stepper.wake.strengths[-1]))
-print("Ratio: %f" % (stepper.wake.strengths[-1] / stepper.wake.circulation))
-s, dgam = compute_gam(body, stepper.bound.vortices)
+print("Ratio of last vortex strength to circulation: %f" % 
+      (stepper.wake.strengths[-1] / stepper.wake.circulation))
+s, gam, dgam = compute_gam(body, stepper.bound.vortices)
 
-s1 = np.linspace(s[-1],1,100)
-dgam_exact = 2 * alpha * np.sqrt((1-s1) / s1)
+pts = body.get_points()
+mid = pts[:-1]+0.5*np.diff(pts, axis=0)
+xmid = mid[:,0]
+rho = 1
+qinf = 0.5*rho*(Uinfty[0]**2+Uinfty[1]**2)
+# See Kuethe and Chow for the following pressure coefficient formula
+cp = -rho*np.linalg.norm(Uinfty)*gam/qinf
+print('xmid',xmid)
+print('cp',cp)
+plt.subplot(211)
+plt.plot(xmid,cp,'x')
+plt.xlabel('Distance along chord')
+plt.ylabel('$C_p$')
+plt.title('Computed coefficient of pressure, AoA = %.1f deg' % alpha_deg)
 
+# Exact solution given section 5.4 of Katz/Plotkin, 2nd Ed.
+s1 = np.linspace(s[-1],1,100)
+dgam_exact = 2 * alpha * np.linalg.norm(Uinfty) * np.sqrt((1-s1) / s1)
+plt.subplot(212)
 plt.plot(s1, dgam_exact, label="thin airfoil theory")
-plt.plot(s, dgam, 'x', label="computed, flat plate")
+plt.plot(s, dgam, 's', label="computed, flat plate")
 plt.xlabel('Arclength s')
-plt.ylabel(r'$\gamma = d\Gamma/ds$')
+plt.ylabel('$\gamma = d\Gamma/ds$')
```

Comment:
Noted for future reference.  Also, I verified that command-line tests ran successfully before submitting my pull request, but afterwards I noticed some Travis CI failures. I haven't looked into that yet and am unsure if it's related to the submitted changes.

> On Nov 22, 2015, at 07:56, Clancy Rowley <notifications@github.com> wrote:
> 
> In scripts/thin_airfoil_unsteady.py:
> 
> >  plt.xlabel('Arclength s')
> > -plt.ylabel(r'$\gamma = d\Gamma/ds$')
> > +plt.ylabel('$\gamma = d\Gamma/ds$')
> Should be a raw string (r'$\gamma...$'). It might not matter here, but if this were, for instance \tau instead of \gamma, then then \t would be interpreted as a tab; similarly, \nu would be interpreted as a newline, followed by the letter u. Always best to use raw strings for LaTeX.
> 
> —
> Reply to this email directly or view it on GitHub.
>